app/css/animator.css

```diff
@@ -101,13 +99,13 @@ a {
 /* ========== SIDEBAR ============== */
 
 #sidebar {
-  position: relative;
-  float: right;
-  color: #d3d3d3;
-  z-index: 2;
-  padding-right: 1em;
-  height: 100%;
-  width: 19em;
+    position: relative;
+    float: right;
+    color: #d3d3d3;
+    z-index: 2;
+    padding-right: 1em;
+    height: 100%;
+    width: 19em;
```

Comment:
*\sigh*  I really need to propose some code style standards.  We should have had those already. In the meantime, please revert this section back to 2 space (trying to keep out unnecessary whitespace changes like this one).